compiler/src/iree/compiler/Codegen/LLVMCPU/KernelDispatch.cpp

```diff
@@ -2407,19 +2427,27 @@ setLoweringConfigForComputeOps(mlir::FunctionOpInterface entryPointFn,
             } else {
               tileSizesList.push_back(zeros);
             }
+            // ATM Assume there are no scalable sizes for reduction dims
+            scalableTileFlagsList.push_back(falseVec);
```

Comment:
Should this be an assertion?

Reply:
Assert would be tricky as we are simply not tracking this information yet. In fact, it will be much easier just to add it here and in other places where it's missing. Let me do that instead.